setup.py

```diff
@@ -39,12 +39,18 @@
 VSC_INSTALL_REQ_VERSION = '0.9.19'
 
 PACKAGE = {
-    'version': '2.4.18',
+    'version': '2.5.0',
```

Comment:
not sure if there's enough here to go to 2.5.0, but on the other hand, I don't really mind (the version number is getting quite weird anyway ;-))